app/views/katello/api/v2/host_collections/_host_collection.json.rabl

```diff
@@ -1,4 +1,5 @@
 attribute :pulp_id, :name, :organization_id, :max_hosts, :description, :total_hosts, :unlimited_hosts
+attributes :host_ids
```

Comment:
@waldenraines, I realize that and I am not opposed to it.  I was just raising that it is technically a regression.  The line added here is the equivalent to the line that was previously removed.  ;)  I plan to move it to 'show' per the suggestion.